pkg/api/plugin.go

```diff
@@ -73,8 +73,8 @@ type PluginConfig struct {
 	SyncImage       string
 	AcceptLanguages []string
 	GenevaConfig    GenevaConfig
-
-	TestConfig TestConfig
+	IsAdmin         bool
```

Comment:
why is this here?  surely it shouldn't be?

Reply:
Hrm, shouldn't be here - I have split validation into its own PR, see https://github.com/openshift/openshift-azure/pull/854